docs/editors/emacs.md

```diff
@@ -116,15 +115,10 @@ To use Metals in Emacs, place this snippet in your Emacs configuration (for exam
 > You may need to disable other packages like `ensime` or sbt server to prevent
 > conflicts with Metals.
 
-If you start Emacs now then it will fail since the `metals-emacs` binary does
-not exist yet.
-
-```scala mdoc:bootstrap:metals-emacs emacs
-
-```
-
-The `-Dmetals.client=emacs` flag is important since it configures Metals for
-usage with Emacs.
+Next you have to install metals server. Emacs can do it for you when `lsp-mode`
+is enabled in a scala buffer or via `lsp-install-server` command. Also you can
+do it manually executing `coursier install metals` and configuring `$PATH`
+variable properly.
```

Comment:
I left a comment on the pr in the other repo, but since using `cs install` doesn't work at all to get the Metals snapshots, it might be better to use bootstrap instead and allow the user to just give a version or default to `latest.stable`. WDYT?

Reply:
I thought that `cs install` is a preferable way to install metals now, as has been said in https://github.com/scalameta/metals/issues/1884 . As for myself I don't use metals snapshots, so not sure how good they are and whether we should provide an easy option to do that :)

Reply:
Ha, I'm probably on the far other end where I only use snapshots, but I'm not the norm there. I'd say that yea if a user is going to install it themselves, then `cs install metals` is by far the easiest way to install ithe recommended way. However, imo if a client is going to abstract over that and just use Coursier under the hood, then using `cs bootstrap` offers a way to also automate the installation and provide a way to install snapshots if they want.

Reply:
This way of installing has one drawback - if something is changed here https://github.com/coursier/apps/blob/f6ad7f3dffd2bf6fddc8991853a32919fd62e44d/apps/resources/metals.json we have to reflect this change in each client, and it easily can remain unnoticed.

Reply:
Should we create an issue for coursier to add `--version` flag to `cs install`? :)

Reply:
Hm, looks like it already can handle versions: https://github.com/coursier/coursier/issues/1249

Reply:
Yes, works like `cs install metals:version`, so I'll add it then.

Reply:
Sorry if I wasn't clear before, but this won't work. That was partly what I was getting at with there is no way to install snapshots with `cs install metals`. For example in the above if you try it locally you get:

```
❯ cs install metals:0.10.2+5-919c430c-SNAPSHOT
https://repo1.maven.org/maven2/io/get-coursier/apps/maven-metadata.xml
  No new update since 2021-03-31 19:34:20
Exception in thread "main" java.lang.Exception: coursier.install.AppArtifacts$ScalaDependenciesNotFound: Can't find a scala version suffix for org.scalameta::metals:0.10.2+5-919c430c-SNAPSHOT (likely a non existing module or version)
        at coursier.install.AppDescriptor.artifacts(AppDescriptor.scala:66)
        at coursier.install.InstallDir.update$1(InstallDir.scala:259)
        at coursier.install.InstallDir.$anonfun$createOrUpdate$21(InstallDir.scala:385)
        at coursier.install.InstallDir.$anonfun$createOrUpdate$21$adapted(InstallDir.scala:384)
        at coursier.install.Updatable$.get$1(Updatable.scala:55)
        at coursier.install.Updatable$.$anonfun$writing$1(Updatable.scala:96)
        at coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:72)
        at coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:98)
        at coursier.install.Updatable$.writing(Updatable.scala:96)
        at coursier.install.InstallDir.createOrUpdate(InstallDir.scala:384)
        at coursier.install.InstallDir.createOrUpdate(InstallDir.scala:87)
        at coursier.cli.install.Install$.$anonfun$run$7(Install.scala:108)
        at coursier.cli.install.Install$.$anonfun$run$7$adapted(Install.scala:95)
        at scala.collection.immutable.List.foreach(List.scala:431)
        at coursier.cli.install.Install$.run(Install.scala:95)
        at coursier.cli.Coursier$.$anonfun$runA$2(Coursier.scala:152)
        at coursier.cli.Coursier$.$anonfun$runA$2$adapted(Coursier.scala:140)
        at coursier.cli.CommandAppPreA.run(CommandAppPreA.scala:22)
        at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$5(CommandAppWithPreCommand.scala:99)
        at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$5$adapted(CommandAppWithPreCommand.scala:99)
        at scala.util.Either.fold(Either.scala:191)
        at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$3(CommandAppWithPreCommand.scala:99)
        at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$3$adapted(CommandAppWithPreCommand.scala:85)
        at scala.Option.foreach(Option.scala:407)
        at caseapp.core.app.CommandAppWithPreCommand.main(CommandAppWithPreCommand.scala:85)
        at coursier.cli.Coursier$.main(Coursier.scala:105)
        at coursier.cli.Coursier.main(Coursier.scala)
Caused by: coursier.install.AppArtifacts$ScalaDependenciesNotFound: Can't find a scala version suffix for org.scalameta::metals:0.10.2+5-919c430c-SNAPSHOT (likely a non existing module or version)
        at coursier.install.AppDescriptor.$anonfun$processDependencies$9(AppDescriptor.scala:184)
        at scala.Option.toRight(Option.scala:485)
        at coursier.install.AppDescriptor.processDependencies(AppDescriptor.scala:184)
        at coursier.install.AppDescriptor.artifacts(AppDescriptor.scala:64)
        ... 26 more
```

Even if you specify the repository you still have issues.

Reply:
In that case above I use Scala 2.13 locally, and Metals isn't published for 2.13, so it won't find the `metals_2.13` artefact.

Reply:
There is also some issues in general when trying to get the latest snapshot as you can see here https://github.com/coursier/coursier/issues/1700

Reply:
That's sad :(